sample.tf

```diff
@@ -10,10 +10,10 @@ terraform {
 
 provider "aws" {
   profile = "default"
-  region  = "us-west-2"
+  region  = "us-west-1"
 }
 
-resource "aws_instance" "app_server" {
+resource "aws_instance" "app_seraver" {
```

Comment:
Issue - EC2 instances should disable IMDS or require IMDSv2 as this can be related to the weaponization phase of kill chain 
 category - Infrastructure Security 
 severity - MEDIUM